cluster_tools/cluster_tools/__init__.py

```diff
@@ -116,8 +116,8 @@ def get_executor(environment: str, **kwargs: Any) -> "Executor":
         return MultiprocessingExecutor(**kwargs)
     elif environment == "sequential":
         return SequentialExecutor(**kwargs)
-    elif environment == "debug_sequential":
-        return DebugSequentialExecutor(**kwargs)
-    elif environment == "test_pickling":
-        return PickleExecutor(**kwargs)
+    elif environment == "sequential_with_pickling":
+        return SequentialPickleExecutor(**kwargs)
+    elif environment == "multiprocessing_with_pickling":
+        return MultiprocessingPickleExecutor(**kwargs)
```

Comment:
It's probably more safe to do so, yes. Especially since I just noticed again, that `debug_sequential` was exposed via the CLI before this PR. I'll re-add them and also add a deprecation warning.

Reply:
See https://github.com/scalableminds/webknossos-libs/pull/1193#discussion_r1858606790